aiodocker/docker.py

```diff
@@ -42,7 +42,23 @@ def __init__(self,
                  connector=None,
                  session=None,
                  ssl_context=None,
-                 api_version='v1.26'):
+                 api_version='v1.26',
+                 loop=None):
```

Comment:
I didn't know that... look what I found on the web when trying to find more information: https://groups.google.com/forum/#!topic/python-tulip/yF9C-rFpiKk